www/docs/en/dev/plugin_ref/spec.md

```diff
@@ -17,700 +17,489 @@ license: >
-        id="com.alunny.foo"
-        version="1.0.2">
+Plugin.xml file defines the structure and settings required for your plugin. It has several elements to provide details about your plugin.
 
-## _plugin_ Element
+## plugin
 
-The `plugin` element is the plugin manifest's top-level element. It
-features the following attributes:
+  The `plugin` element is the plugin manifest's top-level element.
 
-* `xmlns` (required):
-  The plugin namespace, `http://apache.org/cordova/ns/plugins/1.0`. If
-  the document contains XML from other namespaces, such as tags to be
-  added to the `AndroidManifest.xml` file, those namespaces should
-  also be included in the top-level element.
+  Attributes(type) | Description
+  ---------------- | ------------
+  xmlns(string) | *Required* <br/> The plugin namespace, `http://apache.org/cordova/ns/plugins/1.0`. If the document contains XML from other namespaces, such as tags to be added to the `AndroidManifest.xml` file, those namespaces should also be included in the <plugin> element.
+  id(string) | *Required* <br/> A reverse-domain style identifier for the plugin.
+  version(string) | *Required* <br/> A version number for the plugin, that matches the following major-minor-patch style regular expression: `^\d+[.]\d+[.]\d+$`
 
-* `id` (required):
-  A reverse-domain style identifier for the plugin, such as
-  `com.alunny.foo`
+  Example:
+  ```
+  <?xml version="1.0" encoding="UTF-8"?>
+  <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
+      xmlns:android="http://schemas.android.com/apk/res/android"
+      id="com.alunny.foo"
+      version="1.0.2">
+  ```
 
-* `version` (required):
-  A version number for the plugin, that matches the following
-  major-minor-patch style regular expression:
+### engines and engine
 
-        ^\d+[.]\d+[.]\d+$
+  The child elements of the `<engines>` element specify versions of Apache Cordova-based frameworks that this plugin supports. Plugman aborts with a non-zero code for any plugin whose target project does not meet the engine's constraints. If no <engine> tags are specified, plugman attempts to install into the specified cordova project directory blindly.
 
-## _engines_ and _engine_ Elements
+  Attributes(type) | Description
+  ---------------- | ------------
+  name(string) | *Required* <br/> Name of the engine. Here are the default engines that are supported : <ul><li> `cordova` </li> <li> `cordova-plugman` </li> <li> `cordova-android` </li> <li> `cordova-ios` </li> <li> `cordova-blackberry10` </li> <li> `cordova-wp8` </li> <li> `cordova-windows` </li> <li> `android-sdk` // returns the highest Android api level installed </li> <li> `apple-xcode` // returns the xcode version </li> <li> `apple-ios` // returns the highest iOS version installed </li> <li> `apple-osx` // returns the OSX version </li> <li> `blackberry-ndk` // returns the native blackberry SDK version </li> You can also specify a custom framework apart from the default ones.
```

Comment:
The comments here should probably just be put in parentheses. Looks strange in plain text